lib/screens/register_screen.dart

```diff
@@ -0,0 +1,298 @@
+              const SizedBox(height: 5.0),
+              TextFormField(
+                  autocorrect: false,
+                  controller: _lastnameController,
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Last name cannot be empty';
+                    }
+                    return null;
+                  },
+                  decoration: kTextFieldDecoration.copyWith(
+                      hintText: 'Last Name',
+                      prefixIcon: const Icon(
+                        FontAwesomeIcons.user,
+                        color: kAppColor,
+                      ))),
+              const SizedBox(height: 5.0),
+              TextFormField(
+                autocorrect: false,
+                controller: _bioController,
+                validator: (value) {
+                  if (value == null || value.isEmpty) {
+                    return 'Bio cannot be empty';
+                  }
+                  return null;
+                },
+                decoration: kTextFieldDecoration.copyWith(
+                    hintText: 'Bio',
+                    prefixIcon: const Icon(
+                      FontAwesomeIcons.pen,
+                      color: kAppColor,
+                    )),
+              ),
+              const SizedBox(height: 5.0),
+              TextFormField(
+                  autocorrect: false,
+                  controller: _hometownController,
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Hometown cannot be empty';
+                    }
+                    return null;
+                  },
+                  decoration: kTextFieldDecoration.copyWith(
+                      hintText: 'Hometown',
+                      prefixIcon: const Icon(
+                        FontAwesomeIcons.home,
+                        color: kAppColor,
+                      ))),
+              const SizedBox(height: 5.0),
+              TextFormField(
+                  autocorrect: false,
+                  controller: _yearController,
+                  validator: (value) {
+                    if (value == null || value.isEmpty) {
+                      return 'Year cannot be empty';
+                    }
+                    return null;
+                  },
+                  decoration: kTextFieldDecoration.copyWith(
+                      hintText: 'Year of Birth',
+                      prefixIcon: const Icon(
+                        FontAwesomeIcons.birthdayCake,
+                        color: kAppColor,
+                      ))),
+              const SizedBox(height: 5.0),
+              SizedBox(
+                width: double.infinity,
+                child: ElevatedButton(
+                  onPressed: () async {
+                    if (_formKey.currentState!.validate()) {
+                      ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
If you want, I have created a snackbar object that can be used by passing in the parameters 'context, text, length of time shown'. It is under the file lib/constants/snackbar. This would ensure all snackbars across the entire app appear the same way. An example can be seen in lib/screens/login_screen.dart line 92

Reply:
replaced snackbars